apidoc/Titanium/UI/View.yml

```diff
@@ -404,9 +404,15 @@ events:
       - name: force
         summary: |
             The current force value of the touch event.
-            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later.
+            Note: This property is only available for iOS devices that support 3D-Touch and run 9.0 or later and on some android devices
```

Comment:
removed the "some". The Android documentation says you'll get a value. So it should just be 0 on devices that don't feature pressure